### ISSUE TITLE
[BUG] error add file input

### ISSUE THREAD

**Issue by tiotimus** [TiagoSilvaPereira/vemto-issues#75]
```
TypeError: Cannot read property 'validation' of undefined
    at Function.fromFieldTypeSettings (app://./js/chunk-common.f141e59c.js:1:42205)
    at Function.fromFieldDefaultSettings (app://./js/chunk-common.f141e59c.js:1:42115)
    at Function.fromField (app://./js/chunk-common.f141e59c.js:1:41923)
    at Proxy.addInputToCrud (app://./js/chunk-common.f141e59c.js:1:17866)
    at a.create (app://./js/project_crud.2431afb4.js:1:26216)
    at app://./js/project_crud.2431afb4.js:1:26126
```

---

vemto 0.7.3
Electron 8.5.5
win32 10.0.19041
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @tiotimus thank you for reporting... I'll fix this bug in the next Vemto version. Cheers

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.7.7
